.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
```

Comment:
Can you add 3.8?

Reply:
In the azure-pipelines.yml it said "Python 3.8 does not work at the moment." so i did not add 3.8.
But if you say i should add it here i can do that, sure.